code/datums/components/crafting/recipes.dm

```diff
@@ -703,6 +703,13 @@
 				/obj/item/stack/rods = 1)
 	category = CAT_MISC
 
+/datum/crafting_recipe/noose
+	name = "Noose"
+	result = /obj/structure/chair/noose
+	reqs = list(/obj/item/stack/cable_coil = 20)
+	time = 20 //womp womp
```

Comment:
i just realized you did the forbidden trait rework thing, we have the SECONDS and MINUTES defines for numbers that are supposed to represent time
```suggestion
	time = 2 SECONDS //womp womp
```